doc/README.md

```diff
@@ -17,7 +17,7 @@ Contents:
   * [Both `semgrep` and `semgrep-core`](#both-semgrep-and-semgrep-core) 
 * [Development Workflow](#development-workflow)
 
-This README gives an overview of the repository. For further information on building, you will be directed to [SEMGREP_CORE_CONTRIBUTING.md](fds) and/or [SEMGREP_CONTRIBUTING.md](fdsfd) in [Making a Change](#making-a-change). 
+This README gives an overview of the repository. For further information on building, you will be directed to [SEMGREP_CORE_CONTRIBUTING.md](https://github.com/returntocorp/semgrep/blob/develop/doc/SEMGREP_CORE_CONTRIBUTING.md) and/or [SEMGREP_CONTRIBUTING.md](https://github.com/returntocorp/semgrep/blob/develop/doc/SEMGREP_CONTRIBUTING.md) in [Making a Change](#making-a-change). 
```

Comment:
I think you can use 
```
[SEMGREP_CORE_CONTRIBUTING.md](doc/SEMGREP_CORE_CONTRIBUTING.md)
```
and github markdown handles converting it to a link in the repo